schema/alpha/artifacts/executable.json

```diff
@@ -1,6 +1,31 @@
 {
     "$id": "http://github.com/kubeflow/metadata/schema/alpha/artifacts/executable.json",
     "$schema": "http://json-schema.org/draft-07/schema#",
+    "examples": [{
+        "category": "artifact",
+        "id": "1234",
+        "input_type": [
+            {
+                "apiversion": "v1",
+                "category": "artifact",
+                "kind": "data_set",
+                "namespace": "my-company.org"
+            }
+        ],
+        "kind": "my-traning-code",
+        "namespace": "my-company.org",
+        "output_type": [
+            {
+                "apiversion": "v1",
+                "category": "artifact",
+                "kind": "model",
+                "namespace": "my-company.org"
+            }
+        ],
+        "apiversion": "v1",
+        "uri": "file://local_file.py",
```

Comment:
In most real world scenarios, its not a straightforward uri for file location

Reply:
Yes, this can be any string as long as the systems/users know how to use it. [more examples](https://github.com/kubeflow/metadata/blob/master/schema/alpha/docs/artifacts/artifact.md#uri-examples)